opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -219,9 +219,14 @@ def create_time_series_list(self, v_data, option_resource_type,
                 # point.value.distribution_value.range.min = agg_data.min
                 # point.value.distribution_value.range.max = agg_data.max
                 bounds = dist_value.bucket_options.explicit_buckets.bounds
+                # Stackdriver expects an "underflow" bucket for samples in
+                # (-inf, 0), DistributionAggregationData has all positive
+                # values.
+                bounds.extend([0])
                 bounds.extend(list(map(float, agg_data.bounds)))
 
                 buckets = dist_value.bucket_counts
+                buckets.extend([0])
```

Comment:
Better to add comment here also, "The first bucket (underflow bucket) should always be 0 count because the Metrics first bucket is [0, first_bound) but StackDriver distribution consists of an underflow bucket (number 0)."